ddtrace/contrib/unittest/patch.py

```diff
@@ -66,31 +144,128 @@ def _extract_module_name_from_test_method(item):
     return getattr(item, "__module__", None)
 
 
-def _extract_test_skip_reason(item):
+def _extract_module_name_from_module(item):
+    if _is_test(item):
+        return type(item).__module__
+    return ""
+
+
+def _extract_test_reason(item):
     return item[1]
 
 
 def _extract_test_file_name(item):
     return os.path.basename(inspect.getfile(item.__class__))
 
 
-def is_unittest_support_enabled():
```

Comment:
This function is now part of our public api. We can't just remove it. We will need to deprecate it first. Here's an example on how to do that: https://github.com/DataDog/dd-trace-py/commit/c0b64cdc648845918c463155ad1a5fe8ad2dc309. We will also need to add a deprecation release note.

Reply:
Do we need to go through the deprecation process for this one? unittest is in beta right now (before this PR is merged) and behind a feature flag. Do we commit to public APIs in that case?

Reply:
It's not documented anywhere, but anecdotally I believe the answer is no, we do not maintain public APIs for beta products.